docs/reference/commandline/dockerd.md

```diff
@@ -68,6 +68,7 @@ Options:
       --log-opt value                         Default log driver options for containers (default map[])
       --max-concurrent-downloads int          Set the max concurrent downloads for each pull (default 3)
       --max-concurrent-uploads int            Set the max concurrent uploads for each push (default 5)
+      --metrics-addr string                   Set default address and port to serve the metrics api on
```

Comment:
do we need the word 'default'? Some other address&port isn't going to be chosen, right?